Userland/Libraries/LibCore/Timer.cpp

```diff
@@ -13,11 +13,11 @@ Timer::Timer(Object* parent)
 {
 }
 
-Timer::Timer(int interval, Function<void()>&& timeout_handler, Object* parent)
+Timer::Timer(int interval_ms, Function<void()>&& timeout_handler, Object* parent)
```

Comment:
It might make sense to also add an overload that takes `AK::Time`, which stores the time in a canonical format, and lets you easily convert between seconds, milliseconds,  nanoseconds, etc

Reply:
Not necessarily for this PR, just putting it out there...